src/index.html

```diff
@@ -4,13 +4,64 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Search bar Airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar__wrapper">
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        class="search-bar__form"
+      >
+        <label for="searchBarBig" class="search-bar__label big" data-qa="big">
+          <img
+            class="search-bar__img"
+            src="./images/Search.svg"
+            alt="Search Logo"
+          >
+        </label>
+        <input
+          type="search"
+          placeholder="Try “Los Angeles”"
+          autofocus="on"
```

Comment:
remove autofocus, it breaks your tests
![image](https://user-images.githubusercontent.com/37161315/154655085-5453538f-5ffb-4277-9a76-9a0094bcc6d7.png)

```suggestion
```

Reply:
Done.